internal/daemon/controller/controller.go

```diff
@@ -414,13 +416,38 @@ func (c *Controller) AuthorizeNodeeWorker(request string) error {
 	const op = "controller.(Controller).AuthorizeWorker"
```

Comment:
Unsure if the op needs to change or if the function name should
```suggestion
	const op = "controller.(Controller).AuthorizeNodeeWorker"
```